env0/common_template.go

```diff
@@ -9,20 +9,29 @@ import (
 type TemplateType string
 
 const (
-	CustomFlow     TemplateType = "custom flow"
-	ApprovalPolicy TemplateType = "approval policy"
+	CustomFlow     TemplateType = "custom-flow"
+	ApprovalPolicy TemplateType = "approval-policy"
 )
 
 func getTemplate(templateType TemplateType) map[string]*schema.Schema {
+	var text string
+
+	switch templateType {
+	case CustomFlow:
+		text = "custom flow"
+	case ApprovalPolicy:
+		text = "approval-policy"
```

Comment:
why the word "text"? its very general - whats wrong with template type?
btw - in our FE we decided on a convention that both of those template will be called "Configuration Template" because they have some similarities (they are configuration that the user define through a template)
maybe using this term can benefit you in some places

Reply:
maybe even the name of the file could be "configuration_template_something" because you are using it only for custom-flow and approval-policy and not for a real template

Reply:
sure!